fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/resources/BaseAccessControlled.java

```diff
@@ -24,8 +24,6 @@
 abstract class BaseAccessControlled<R extends BaseAccessControlled>
     implements Resource.AccessControlled {
 
-  abstract R setPermissions(Permissions p);
```

Comment:
moved this to the AccessControlled interface (all the concrete subclasses already explicitly support it)